R/derive_param_tte.R

```diff
@@ -529,7 +600,7 @@ derive_param_tte <- function(dataset = NULL,
 #' @return A dataset with one observation per subject as described in the
 #'   "Details" section.
 #'
-#' @noRd
+#' @keywords internal
```

Comment:
this update is causing the R CMD check fail in the CI - not sure if it was needed or what the rationale for updating it was? Maybe @bundfussr could comment

Reply:
@rossfarrugia , this was changed to create the documentation for the function. Before no help or html page was created.

@ProfessorP-beep , as the function is not exported, you need to prefix it in the example with `admiral:::`.